vro-types/o11n-plugin-aria/index.d.ts

```diff
@@ -10,16 +10,429 @@ declare class VraHost {
 	readonly sessionMode: string;
 	readonly connectionType: string;
 	readonly user: string;
+	/**
+	 * Automation Host No Argument Constructor.
+	 */
+	constructor();
+
+	createInfrastructureClient(): VraInfrastructureClient;
+
 	/**
 	 * Validates the Host Connection.
 	 */
 	validate(): boolean;
 
+	/**
+	 * Destroys the connection object
+	 */
+	destroy(): void;
+
 	/**
 	 * Creates a generic REST client for Automation Host
 	 */
 	createRestClient(): VraGenericRestClient;
+}
+
+declare class VraInfrastructureClient {
+	constructor();
+	/**
+	 * @param genericRestClient
+	 */
+	constructor(genericRestClient: VraGenericRestClient);
+
+	/**
+	 * Create Data Collector service to invoke APIs.
+	 */
+	createDataCollectorService(): VraDataCollectorService;
+
+	/**
+	 * Create a Request service to invoke APIs.
+	 */
+	createRequestService(): VraRequestService;
+
+	/**
+	 * Create Cloud Zone service to invoke APIs.
+	 */
+	createCloudZoneService(): VraCloudZoneService;
+
+	/**
+	 * Create a Cloud Account service to invoke APIs.
+	 */
+	createCloudAccountService(): VraCloudAccountService;
+}
+
+declare class VraCloudAccountService {
+	constructor();
+	constructor();
+
+	/**
+	 * @param genericRestClient
+	 */
+	setGenericRestClient(genericRestClient: VraGenericRestClient): void;
+
+	/**
+	 * Delete a vSphere Cloud Account with a given id
+	 * @param cloudAccountVsphereId
+	 */
+	deleteVSphereCloudAccountById(cloudAccountVsphereId): void;
+
+	/**
+	 * Delete a vSphere Cloud Account with a given cloudAccountVsphere object
+	 * @param cloudAccountVsphere
+	 */
+	deleteVSphereCloudAccount(
+		cloudAccountVsphere: VraCloudAccountVsphere
+	): void;
+
+	/**
+	 * Create a vSphere cloud account
+	 * @param body
+	 */
+	createVSphereCloudAccount(
+		body: VraCloudAccountVsphereSpecification
+	): VraCloudAccountVsphere;
+
+	/**
+	 * Create a vSphere cloud account in an asynchronous manner
+	 * @param body
+	 */
+	createVSphereCloudAccountAsync(
+		body: VraCloudAccountVsphereSpecification
+	): VraRequestTracker;
+
+	/**
+	 * Update vSphere cloud account in an asynchronous manner
+	 * @param cloudAccountVsphere
+	 * @param body
+	 */
+	updateVSphereCloudAccountAsync(
+		cloudAccountVsphere: VraCloudAccountVsphere,
+		body: VraUpdateCloudAccountVsphereSpecification
+	): VraRequestTracker;
+
+	/**
+	 * Update vSphere cloud account
+	 * @param cloudAccountVsphere
+	 * @param body
+	 */
+	updateVSphereCloudAccount(
+		cloudAccountVsphere: VraCloudAccountVsphere,
+		body: VraUpdateCloudAccountVsphereSpecification
+	): VraCloudAccountVsphere;
+}
+
+declare class VraUpdateCloudAccountVsphereSpecification {
+	hostName: string;
+	acceptSelfSignedCertificate: boolean;
+	createDefaultZones: boolean;
+	password: string;
+	associatedCloudAccountIds: Object[];
+	regions: Object[];
+	dcid: string;
+	name: string;
+	description: string;
+	username: string;
+	tags: Object[];
+	constructor();
+	constructor();
```

Comment:
I think this one is duplicate

Reply:
Resolved as well. All duplicates were removed.